frontend/src/components/mine/Summary/MineSummary.js

```diff
@@ -94,14 +94,24 @@ export const MineSummary = (props) => {
             <Row gutter={16} type="flex">
               {props.partyRelationships
                 .filter(isActive)
-                .filter((pr) => ["MMG", "PMT"].includes(pr.mine_party_appt_type_code))
+                .filter((pr) => pr.mine_party_appt_type_code === "MMG")
                 .map((partyRelationship) =>
                   renderPartyRelationship(
                     props.mine,
                     partyRelationship,
                     props.partyRelationshipTypes
                   )
                 )}
+              {props.mine.mine_permit.map((permit) =>
+                renderPartyRelationship(
+                  props.mine,
+                  props.partyRelationships
+                    .filter(isActive)
+                    .filter((pr) => pr.mine_party_appt_type_code === "PMT")
+                    .filter((pr) => pr.related_guid === permit.permit_guid)[0],
```

Comment:
I doubt it makes much/any difference with the amount of data that we're dealing with in this part of the code, but it could be worth mentioning since it's part of the rendered JSX.. these chained filters are creating unnecessary iterations through the array.

These could be composed into a single conditional, and perhaps assigned to a variable for readability.
```javascript
const aGoodConditionName = (pr) => isActive(pr) &&
  pr.mine_party_appt_type_code === "PMT" && 
  pr.related_guid === permit.permit_guid

// stuff
props.partyRelationships
                    .filter(aGoodConditionName)[0]

```

Reply:
That crossed my mind as well, but I think Permittee import logic may change next sprint (MDS-1516) as a different source for the "current" permittee should be considered and I'm not sure yet how that will be represented in the collection of Permittees.

Reply:
Actually, let's pull it out anyways even if the logic changes. Might as well and I have the file open.

Reply:
Updated so that each only has one filter 👍